### PR TITLE
(maint) Reordering broke puppetdb.conf templating

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

**Comment by Xtigyro** [comment]
That's a very important fix!

**Comment by Xtigyro** [comment]
Confirmed - the PR also fixes the booting of Puppet Server in the Helm chart. It starts perfectly now:
```shell
/repos/xtigyro/puppetserver-helm-chart # klpp puppetserver-puppetserver-helm-cha-puppetserver-699fbfc99cmgvmr
Running /docker-entrypoint.d/10-analytics.sh
(/docker-entrypoint.d/10-analytics.sh) Pupperware analytics disabled; skipping metric submission
Running /docker-entrypoint.d/20-use-templates-initially.sh
Loading new empty VOLUME with default /opt/puppetlabs/server/data/puppetserver content (LCOW moby 39892 workaround)
Running /docker-entrypoint.d/30-set-permissions.sh
Running /docker-entrypoint.d/40-update-puppetdb-conf.sh
Running /docker-entrypoint.d/50-set-certname.sh
```

File: docker/puppetserver/Dockerfile [hunk 1]
```diff
@@ -66,6 +66,6 @@ COPY docker/puppetserver/logback.xml \
      docker/puppetserver/request-logging.xml \
      /etc/puppetlabs/puppetserver/
 COPY docker/puppetserver/puppetserver.conf /etc/puppetlabs/puppetserver/conf.d/
-COPY docker/puppetserver/puppetdb.conf /etc/puppetlabs/puppet/
```
Comment (Iristyle) [inline docker/puppetserver/Dockerfile]: Bug introduced in https://github.com/underscorgan/puppetserver/commit/361a9a155b6a2a532aab403d055faf3ebddcc185#diff-0bc70a0d801716e2c729708b9e6015baR58

Comment (Iristyle) [reply docker/puppetserver/Dockerfile]: Sorry about that everyone!

**Pull request merged**